index.html

```diff
@@ -38,95 +44,112 @@ <h1 class="greet">Hey There. <br> I’m Alex</h1>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art3">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art4">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
-    </div>
-
-    <div class="professional-art art5">
-      <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
-      <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="professional-list">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="prof-btn">See Project</button>
+      <div class="line"></div>
     </div>
+    <div class="work-container">
+      <div class="multi-post">
+        <img src="icons/images/Img Placeholder1.png" alt="illustration image">
+        <div>
+          <h2 id="multi-head-two">Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="multi-btn">See Project</button>
+        </div>
+      </div>
+  
+      <div class="professional-art art1 prt">
+        <h2 class="prof-header">Profesional Art <br> Printing Data</h2>
+        <p class="multi-paragraph-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="professional-list">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button class="prof-btn">See Project</button>
+  
+      </div>
```

Comment:
You did a great job adding a black border when hovering over the cards but as the previous reviewer requested, please fix the default and hover state of the cards. 👨‍🎨
<img width="544" alt="Screenshot 2023-01-20 at 8 26 29 AM" src="https://user-images.githubusercontent.com/25887253/213624543-7932075e-3b62-4047-b1c1-9dd7f6895849.png">